backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/directory/FileSystemAccessStrategyName.java

```diff
@@ -15,11 +15,6 @@
 public enum FileSystemAccessStrategyName {
 
 	AUTO( "auto" ),
-	/**
-	 * @deprecated This strategy will be removed in a future version of Lucene.
-	 */
-	@Deprecated
-	SIMPLE( "simple" ),
```

Comment:
Can you please also remove the corresponding entry from the reference documentation, if any?